soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -3130,11 +3414,20 @@ void DrawRandoEditor(bool& open) {
                                         "expected to be collected after getting Sun's Song.");
                     PaddedSeparator();
 
-                    // Shuffle Cows
-                    SohImGui::EnhancementCheckbox(Settings::ShuffleCows.GetName().c_str(), "gRandomizeShuffleCows");
-                    InsertHelpHoverText(
-                        "Cows give a randomized item from the pool upon performing Epona's Song in front of them.");
-                    PaddedSeparator();
+                        // Shuffle Scrubs
+                        ImGui::Text(Settings::Scrubsanity.GetName().c_str());
+                        InsertHelpHoverText(
+                            "Off - Scrubs will not be shuffled.\n"
+                            "\n"
+                            "Affordable - Scrubs will be shuffled and their item will cost 10 rupees.\n"
+                        );
+                        SohImGui::EnhancementCombobox("gRandomizeShuffleScrubs", randoShuffleScrubs, 4, 0);
+                        PaddedSeparator();
+
+                        // Shuffle Cows
+                        SohImGui::EnhancementCheckbox(Settings::ShuffleCows.GetName().c_str(), "gRandomizeShuffleCows");
+                        InsertHelpHoverText("Cows give a randomized item from the pool upon performing Epona's Song in front of them.");
+                        PaddedSeparator();
 
```

Comment:
Indentation is off, and it seems like you're missing text in the tooltip for the "Expensive" and "Random Prices" options. I assume you made the tooltip before you made the other options possible.

Also, feel free to move the scrubsanity dropdown above tokensanity. That way we have all the dropdowns grouped up together.